src/BinSkim.Driver/AnalyzeCommand.cs

```diff
@@ -90,10 +90,26 @@
~            {
~                result = base.Run(analyzeOptions);
~            }
~            catch (Exception e)
             {
                 Console.WriteLine(e);
             }
+
+            try
```

Comment:
Is it possible that current run fails no sarif file generated, but the same sarif file from previous run exists and it send the data from previous run to AppInsight?

Reply:
it is, but we would not know.